docker/colony-cdapp-dev-env-block-ingestor

```diff
@@ -1,6 +1,6 @@
 FROM colony-cdapp-dev-env/base:latest
 
-ENV BLOCK_INGESTOR_HASH=bbad0279d810d2088c09c726fc8396f732ca1db5
+ENV BLOCK_INGESTOR_HASH=e082fe153e0f91a5df53d55bc01597a900f44760
```

Comment:
Heads up if updating to the latest that colony network released `glwss3` which includes colony `v12`. I'm not sure the current release of colonyJS `v5` supports this yet

Reply:
Thanks, this is only the ingestor image - as far as I'm aware we kept the network hash the same for now